openid-4-verifiable-presentations-1_0.md

```diff
@@ -549,11 +555,250 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
+The value MUST be a string that specifies the name of the claim within the provided namespace
+in the Verifiable Credential, e.g., `first_name`.
+
+`value`:
+: OPTIONAL. A string, integer or boolean value that specifies the expected value of the claim. If the
+`value` property is present, the Wallet MUST return the claim only if the type and value
+of the claim match the type and value specified in the query.
+
+`values`:
+: OPTIONAL. An array of strings, integers or boolean values that specifies the expected values of the claim.
+If the `values` property is present, the Wallet MUST return the claim only if the
+type and value of the claim both match for at least one of the elements in the array.
+
+`alternative_claims`:
+: OPTIONAL. An array of claim group identifiers that specify alternative groups of claims
+that MAY be returned instead of the claim specified by the `path` or `claim_name` property if
+that claim is not present or does not match the restriction expressed by `value` or `values`. Any
+claim in a group listed in `alternatives` MUST NOT itself contain an `alternatives` property.
+
+### Semantics of `alternative_claims` and `alternative_credentials`
+
+By default, each element within `expect_credentials` or `expect_claims` MUST be returned, unless:
+
+- The element contains an `alternative_claims` or `alternative_credentials`
+  property and for at least one of the groups listed as an alternative, all
+  elements in the group can be returned.
+- The element is part of a group that is itself listed as an alternative in
+  another element and returning the element is not needed to fulfill the
+  previous rule.
+
+Groups of claims are scoped to the specific credential; i.e., groups with the
+same identifier that appear in separate Credential Query in `expect_credentials`
+have no relation to each other.
+
+## Format-specific Properties {#format_specific_properties}
+
+The following format-specific properties are defined:
+
+### Format `vc+sd-jwt` {#format_vc_sd_jwt}
+
+`vct_values`:
+: OPTIONAL. An array of strings that specifies allowed values for
+the type of the requested Verifiable Credential. All elements in the array MUST
+be valid type identifiers as defined in [@!I-D.ietf-oauth-sd-jwt-vc]. The Wallet
+MAY return credentials that inherit from any of the specified types, following
+the inheritance logic defined in [@!I-D.ietf-oauth-sd-jwt-vc].
+
+`credential_signing_alg_values`:
+: OPTIONAL. An array of strings that specifies
+the allowed algorithms for signing the SD-JWT (i.e., only the issuer-signed part).
+
+`kbjwt_signing_alg_values`:
+: OPTIONAL. An array of strings that specifies the
+allowed algorithms for signing the Key-Binding JWT (KB-JWT) (i.e., the part signed
+by the holder).
+
+`sd_alg_values`:
+: OPTIONAL. An array of strings that specifies the allowed hash algorithms for
+digests in the SD-JWT and KB-JWT.
+
+### Format `mso_mdoc` {#format_mso_mdoc}
+
+TBD
```

Comment:
I got a very bad reaction from @bc-pi against the use of TBD in the specs, even if only draft. This is educating me.